bedrock/firefox/forms.py

```diff
@@ -10,8 +10,6 @@
 
 from lib.l10n_utils.dotlang import _lazy as _
 
-LANG_FILES = ['firefox/whatsnew-fx37']
-
```

Comment:
Not specific to this PR (as `firefox/whatsnew-fx37.lang` is already gone), but just checking that the string below ("Sorry. This number isn't...") is still able to be translated.

After a bit of testing, it looks like the `invalid` error message below is never used. I see a similar error message [in the `send_to_device` macro](https://github.com/mozilla/bedrock/blob/master/bedrock/base/templates/macros.html#L606). Does that error message render the one below moot? Not really clear if we should remove the one below, or maybe just add a comment stating that it's never displayed?

Or I could just be confused.